### PR TITLE
#10947 and #10948 improve robosaur folder structure

### PR DESCRIPTION
**Pull request by daniel-jeremia**
* Changed config folder to sample as using the name `config` as an example folder is quite confusing

### CODE REVIEW

**Comment by darwin-gautalius** [comment]
The more I look at it, the more `quickstart` looks like another sample @@"

**Comment by daniel-jeremia** [comment]
> The more I look at it, the more `quickstart` looks like another sample config, but under a different folder @@"

that's true xD

File: quickstart/config/quickstart.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   },
   "documents": {
     "source": "local",
-    "path": "quickstart/create/documents"
+    "path": "quickstart/create/token-based/documents"
```
Comment (darwin-gautalius) [inline quickstart/config/quickstart.json:21]: After seeing the updated structure, I think it discovers other inconsistencies in the `quickstart` folder.
How about making each config under the same folder as its documents?
The initial structure without row-based looks quite makes sense:

```
/quickstart <-- looks like root for token-based
  /config
  /create
  /export
```

however, as the row-based come, the structure, become off:
```
/quickstart
  /config
    /quickstart.json
    /quickstart.row.json
  /create
    /assignment
      /assignment.json <-- `config` but not in `config` folder? 🤔 
    /token-based/documents
    /row-based
      /documents
      /question-set.json <-- `config` but not in `config` folder? 🤔 
  /export
    ??? <-- i'm not sure what the structure will be like
```

not sure yet, but I think we actually need to restructure it like this:
```
/quickstart
  /token-based
    /config
      /config.json
      /assignment.json
      /label-set.json
    /create
      /Project 1
      /Project ...
      /Project N
    /export
      /Project 1
      /Project ...
      /Project N
  /row-based
    /config
      /config.json
      /assignment.json
      /question-set.json
    /create
      /Project 1
      /Project ...
      /Project N
    /export
      /Project 1
      /Project ...
      /Project N
```
with this structure IMO the files are more organized. for users concerning `token-based` only, they have the entire `token-based` folder to be concerned.

**Comment by darwin-gautalius** [comment]
> > The more I look at it, the more `quickstart` looks like another sample config, but under a different folder @@"
> 
> that's true xD

:psyduck2:

File: quickstart/config/quickstart.json (hunk 1 above)
Comment (daniel-jeremia) [reply quickstart/config/quickstart.json:21]: got it, i agree with this

File: quickstart/row-based/config/config.json [hunk 2]
```diff
@@ -1,31 +1,31 @@
 {
   "datasaur": {
-    "host": "https://datasaur.ai",
-    "clientId": "<DATASAUR_CLIENT_ID>",
-    "clientSecret": "<DATASAUR_CLIENT_SECRET>"
+    "host": "http://localhost:8080",
+    "clientId": "9cb9bc9e-96f6-46d1-bbdb-3f42ad5eb449",
```
Comment (darwin-gautalius) [inline quickstart/row-based/config/config.json:4]: oops

**Comment by darwin-gautalius** [comment]
I think quickstart link s should be updated

File: quickstart/row-based/config/config.json [hunk 3]
```diff
@@ -10,22 +10,22 @@
   },
   "export": {
     "source": "local",
-    "prefix": "quickstart/export",
-    "teamId": "<TEAM_ID>",
+    "prefix": "quickstart/row-based/export",
+    "teamId": "1",
```
Comment (darwin-gautalius) [inline quickstart/row-based/config/config.json:14]: sorry, didn't notice this before. should be reverted back to <TEAM_ID>

**Review by darwin-gautalius — APPROVED** [review]

**Pull request merged**